Makefile

```diff
@@ -39,6 +39,8 @@ DOCKER_BUILD = docker build --no-cache \
 	    PLATFORM=$(PLATFORM) \
 	    BUILD_NUMBER=$(BUILD_NUMBER) \
 	    ENABLE_DHCP_GRAPH_SERVICE=$(ENABLE_DHCP_GRAPH_SERVICE) \
+	    PASSWORD=$(PASSWORD) \
```

Comment:
I think you misunderstood marian's comment. What is happening here is that adding this statement will override slave.mk's $PASSWORD with Makefile's $PASSWORD, even if it is empty. So we DO need to check empty before passing it or the value in rule.cofig will be overridden.

What is different here from ENABLE_DHCP_GRAPH_SERVICE is that in the case of ENABLE_DHCP_GRAPH_SERVICE actually empty is a valid value (and also the default value), while for $PASSWORD it is not.